lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -489,7 +489,15 @@ tokenize([T | Rest], Line, Column, Scope, Tokens) when ?pipe_op(T) ->
 % Non-operator Atoms
 
 tokenize([$:, H | T] = Original, Line, Column, Scope, Tokens) when ?is_quote(H) ->
-  case elixir_interpolation:extract(Line, Column + 2, Scope, true, T, H) of
+  OuterScope = case H == $' of
```

Comment:
Instead of changing all other `Scope`s in this function, change the `Scope` above to `BaseScope` and have this one `Scope`. Fewer changes necessary. :)